### DISCUSSION TITLE
Cell voltage reporting

### DISCUSSION THREAD

**Discussion by arumdev** [KG-Development/SmartBMSUtility#14] [Ideas]
I would love to see a delta value to be able to know the overall imbalance of the cells without having to do mental arithmetic every time. It would also be really great of the Lowest cellvoltage and Highest cellvoltage values would indicate *which* cell they refer to in realtime (with no higher than 1 second refresh to make it readable).

Finally, some custom definable notification based alerts based on battery/cell voltage and delta would be great, buta that’s a luxury really…

**Comment by NeariX67** [comment]
Hey, thanks for you recommendations and ideas. But sadly, i will no longer update the "version 1" of my App. I planned to start working on a version 2 that will be written in a newer way, that would support way more iOS 13+ features, but i just can't find the time and motivation to start that journey. If I get to it, i'll happily include your suggestions. Cheers

**Reply by arumdev** [reply]
ok fair enough, I feared that might be the case.

**Comment by TheDodgersOnline** [comment]
Hi

Until this app is upgraded you might consider the XiaoXiangBMS which does everything you asked for here. Screenshot of mine

Regards
